extensions/youtube/src/lib/youtubeapi.ts

```diff
@@ -139,6 +144,11 @@ export interface Video {
   channelTitle: string;
 }
 
+export interface VideoActionProps {
```

Comment:
Since the props interfaces are not part of the business logic of the youtube API please move it to it's own file

Reply:
Moved to `video.tsx`